### DISCUSSION TITLE
如何下载hdr画质的视频

### DISCUSSION THREAD

**Discussion by sino1641** [the1812/Bilibili-Evolved#1946] [提问 / Q&A]
Q:请问如何下载hdr画质的视频？
1. 使用旧版edge打开，能选择真彩HDR选项。
2. 使用Chrome91打开，播放器中没有HDR选项，
侧边栏中打开下载视频按钮所能选择的最高画质只到4K。

TAT校园网网速太差，只能使用下载后观看的workaround
望解答，感谢！

**Comment by xingxing404** [comment]
同希望能添加hdr下载功能，或者弄一个chrome可以看hdr的功能。
